specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2019-08-01/imds.json

```diff
@@ -37,6 +37,10 @@
               "$ref": "#/definitions/GalleryApplications"
             }
           },
+          "204": {
```

Comment:
Why not return a gallery application with an empty applications list? I assume that this is what happened in previous API version. This change introduces friction for clients that want to adopt the new API version.

Reply:
This is correcting a previously incorrect Swagger. The 204 response was always a part of the API.

I'm unsure. IMDS has several providers that are maintained by other teams. In this case, the Gallery Application team would be best suited to answer that question. @koallison what do you think of returning an empty applications list?

Reply:
The Gallery Application API is new, so there aren't any users of Gallery Application using old API versions.